.nuspec/Microsoft.Maui.Controls.SingleProject.targets

```diff
@@ -94,6 +93,23 @@
 
   </Target>
 
+  <!--
+    Make sure the properties are valid before validating. Instead of an error when the user
+    toggles between modes, just handle it and remove the AppxManifest item.
+  -->
+  <Target Name="_ValidateWindowsPackageTypeBeforeTarget"
+          BeforeTargets="_ValidateWindowsPackageType"
+          DependsOnTargets="_MauiRemoveRemoveAppxManifestForUnpackaged" />
+
+  <Target Name="_MauiRemoveRemoveAppxManifestForUnpackaged"
+          Condition="'$(WindowsPackageType)' == 'None' and '@(AppxManifest)' != ''">
+
+    <ItemGroup>
+      <AppxManifest Remove="@(AppxManifest)" />
+    </ItemGroup>
+
+  </Target>
```

Comment:
The issue that #5896 fixes is when the manifest AND unpackaged is specified. That PR tried to fix it by not including the manifest, however, this is wrong because the way the PR did it was to check after the nuget had already defaulted to None and thus never included it.

Technically, this is actually an invalid state with unpackaged AND a manifest, however since we auto-include we also need to un-include. And secondly, what if this is a multi-mode project? Instead of requiring the user to do weird conditions, just not exclude it for the user.